pkg/builder/builder.go

```diff
@@ -862,12 +884,24 @@ func makeScrutinizeInitContainer(vscr *v1beta1.VerticaScrutinize, vdb *vapi.Vert
 		Resources:    vscr.Spec.Resources,
 		Env:          buildCommonEnvVars(vdb),
 	}
-	cnt.Env = append(cnt.Env, append(buildNMATLSCertsEnvVars(vdb),
-		buildScrutinizeTarballEnvVar(tarballName))...)
 	if vdb.Spec.PasswordSecret != "" {
-		cnt.Env = append(cnt.Env, buildScrutinizeDBPasswordEnvVars(
-			names.GenNamespacedName(vscr, vdb.Spec.PasswordSecret))...)
+		if secrets.IsK8sSecret(vdb.Spec.PasswordSecret) {
+			// we mount the password into the scrutinize init container
+			// only when the password secret in on k8s
+			cnt.VolumeMounts = append(cnt.VolumeMounts, corev1.VolumeMount{
+				Name:      scrutinizeDBpasswordMountName,
+				MountPath: paths.ScrutinizeDBpasswordPath,
+			})
+		} else {
+			// the password secret env vars are needed only when the secret
```

Comment:
We should make sure that QA verifies this part that we can still run scrutinize if the dbadmin password is stored in AWS or GSM.